tonic/src/request.rs

```diff
@@ -1,15 +1,15 @@
 use crate::metadata::{MetadataMap, MetadataValue};
 #[cfg(feature = "server")]
 use crate::transport::server::TcpConnectInfo;
-#[cfg(all(feature = "server", feature = "tls"))]
+#[cfg(all(feature = "server", any(feature = "tls", feature = "tls-aws-lc")))]
```

Comment:
I think we should add some lowest-common-denominator feature that replaces `tls` (maybe `tls-any`) so that we don't need the proliferation of `any()`s all over the place.